js/auth.js

```diff
@@ -1,10 +1,12 @@
 const btnsTogglePw = document.querySelectorAll(".btn-toggle");
+const inputs = document.querySelectorAll(".input-area > input");
 
 /**
  * 비밀번호와 비밀번호 확인의 type을 변경하는 함수
+ * @param {*} e
  */
 function changeTypePw(e) {
-  const inputPw = e.target.previousElementSibling; // 버튼 이전 요소, 즉 비밀번호 | 비밀번호 확인 input 요소
+  const inputPw = e.target.closest(".password").querySelector(".pw"); // 비밀번호 | 비밀번호 확인 input 요소
   const type = inputPw.getAttribute("type"); // 비밀번호 | 비밀번호 확인 input의 type 속성
 
   if (type === "password") {
```

Comment:
if 문도 좋지만 최근 강의에서 배운 삼항연산자를 사용하면 코드를 한줄로 줄일 수 있겠네요!
```suggestion
inputPw.setAttribute("type", type === "password" ? "text" : "password");  
```